packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -0,0 +1,268 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const error = require('../error');
+const isA = require('joi');
+const ScopeSet = require('fxa-shared').oauth.scopes;
+
+module.exports = (log, db, config, customs, oauthdb, subhub) => {
+  const totpUtils = require('./utils/totp')(log, config, db);
+
+  const SUBSCRIPTIONS_MANAGEMENT_SCOPE =
+    'https://identity.mozilla.com/apps/subscriptions';
```

Comment:
I *think* I read somewhere that we want scopes to be in URL form going forward. So, I made this one up for the subscription management pages. Is this the right thing to do?

Reply:
Well, this scope is specifically for access to the subscriptions CRUD methods. But, on the payment pages we also want to read profile details for display. So, I figured we'd need both here.